README.md

```diff
@@ -128,6 +124,58 @@ fn main() {
 }
 ```
 
+**Deserializes Strings into Rust Types**:
+
+> For now you have to implement the conversion yourself with the `Deserialize` trait. Soon you'll be able to have that implemented for you via `edn-derive` crate.
+ ```rust
+use edn_rs::{Deserialize, Edn, EdnError};
+
+#[derive(Debug, PartialEq)]
+struct Person {
+    name: String,
+    age: usize,
+}
+
+impl Deserialize for Person {
+    fn deserialize(edn: Edn) -> Result<Self, EdnError> {
+        Ok(Self {
+            name: edn[":name"].to_string(),
+            age: edn[":age"].to_uint().ok_or_else(|| {
+                EdnError::Deserialize("couldn't convert `:age` into `uint`".to_string())
+            })?,
```

Comment:
I think there is a small problem when we developed, that the main type for integer is isize and no usize, just something to review.

Reply:
But I think isize should be default, since we don't know if the user is using negatives or not.